src/app/dataset-view/dataset-view-routing.ts

```diff
@@ -23,6 +23,20 @@ import { DatasetViewComponent } from "./dataset-view.component";
 import { datasetViewResolverFn } from "./resolvers/dataset-view.resolver";
 import { datasetViewActiveTabResolverFn } from "./resolvers/dataset-view-active-tab.resolver";
 import { OverviewComponent } from "./additional-components/overview-component/overview.component";
+import { MetadataSchemaTabComponent } from "./additional-components/metadata-component/tabs/metadata-schema-tab/metadata-schema-tab.component";
+import { MetadataTabs } from "./additional-components/metadata-component/metadata.constants";
+import { MetadataPollingSourceTabComponent } from "./additional-components/metadata-component/tabs/metadata-polling-source-tab/metadata-polling-source-tab.component";
+import { metadataActiveTabResolverFn } from "./additional-components/metadata-component/resolver/metadata-active-tab.resolver";
+import { MetadataLicenseTabComponent } from "./additional-components/metadata-component/tabs/metadata-license-tab/metadata-license-tab.component";
+import { MetadataWatermarkTabComponent } from "./additional-components/metadata-component/tabs/metadata-watermark-tab/metadata-watermark-tab.component";
+import { MetadataTransformationTabComponent } from "./additional-components/metadata-component/tabs/metadata-transformation-tab/metadata-transformation-tab.component";
+import { MetadataPushSourcesTabComponent } from "./additional-components/metadata-component/tabs/metadata-push-sources-tab/metadata-push-sources-tab.component";
+import { metadataSchemaTabResolverFn } from "./additional-components/metadata-component/tabs/metadata-schema-tab/resolvers/metadata-schema-tab.resolver";
+import { metadataWatermarkTabResolverFn } from "./additional-components/metadata-component/tabs/metadata-watermark-tab/resolvers/metadata-watermark-tab.resolver";
+import { metadataTransformationTabResolverFn } from "./additional-components/metadata-component/tabs/metadata-transformation-tab/resolvers/metadata-transformation-tab.resolver";
+import { metadataLicenseTabResolverFn } from "./additional-components/metadata-component/tabs/metadata-license-tab/resolvers/metadata-license-tab.resolver";
+import { metadataPollingSourceTabResolverFn } from "./additional-components/metadata-component/tabs/metadata-polling-source-tab/resolvers/metadata-polling-source-tab.resolver";
+import { metadataPushSourcesTabResolverFn } from "./additional-components/metadata-component/tabs/metadata-push-sources-tab/resolvers/metadata-push-sources-tab.resolver";
```

Comment:
Avoid these imports and implementing child routes this level, as it will push the components into the parent bundle. Instead, follow the pattern of Settings tab:

```
            {
                path: DatasetViewTypeEnum.Settings,
                runGuardsAndResolvers: "always",
                canActivate: [AuthenticatedGuard],
                data: {
                    [ProjectLinks.URL_PARAM_TAB]: DatasetViewTypeEnum.Settings,
                },
                loadChildren: () =>
                    import(
                        /* webpackChunkName: "dataset-view-settings" */
                        "./additional-components/dataset-settings-component/dataset-settings-routing"
                    ).then((m) => m.DATASET_SETTINGS_ROUTES),
            },
```

Reply:
Fixed